### DISCUSSION TITLE
docs/react/fetching-data

### DISCUSSION THREAD

**Discussion by giscus** [gqty-dev/gqty#1016] [Q&A]
# docs/react/fetching-data

a GraphQL client built for rapid iteration.

https://gqty.dev/docs/react/fetching-data

**Comment by mikolajkniejski** [comment]
Hi! I'm a newbie, so forgive if this is something obvious. Is cache shared between different queries? Let's say I want to query some data every two seconds and I need to fetch it in some components deep in the tree. I can not pass it down, because it would make the whole tree rerender every two seconds. Can I use let's say lazy query and run it in the root every two seconds and somehow get the data from it deeper in the three?

**Reply by vicary** [reply]
It is shared, the lazy query will work as described.
